boards/arm/gd32e507v_start/board.cmake

```diff
@@ -1,6 +1,8 @@
 # Copyright (c) 2022, Teslabs Engineering S.L.
 # SPDX-License-Identifier: Apache-2.0
 
+board_runner_args(pyocd "--target=GD32E507VE")
```

Comment:
pyOCD handles target name in lower case.
Convert to lower If target name given as upper case.
Thus, It is correct but should be better to write in lower case.

```
$ pyocd list --target
  Name                      Vendor                  Part Number                  Families                    Source   
----------------------------------------------------------------------------------------------------------------------

~~ snip ~~

  gd32f403rc                GigaDevice              GD32F403RC                   GD32F4xx Series, GD32F403   pack     
  gd32f403re                GigaDevice              GD32F403RE                   GD32F4xx Series, GD32F403   pack    
```

Reply:
done, thanks